transform/mattermost-analytics/models/staging/mm_telemetry_prod/_mm_telemetry_prod__sources.yml

```diff
@@ -0,0 +1,214 @@
+version: 2
+
+sources:
+  - name: mm_telemetry_prod
+    database: 'RAW'
+    schema: mm_telemetry_prod
+    loader: Rudderstack
+    description: |
+      Mattermost server and webapp telemetry data. Stored using Rudderstack. Rudderstack documentations offers an
```

Comment:
@ifoukarakis , wonder if we need to make it clear here or somewhere else that we only capture "more recent" events (Rudderstack) and not the older ones (Segment). Drawing the line at the product version where Rudderstack has been added (v5.0?) might convey the same.